### ISSUE TITLE
[BUG] Tool adds "" to the characters in direct method request payload

### ISSUE THREAD

**Issue by xiongyu0523** [Azure/azure-iot-explorer#439]
**Describe the bug**
If I just input a string like **hello world** in the payload diagram, the device get **"hello world"**.  
If I just input a number like **123** in the payload diagram, the device get just **123**.  
I'm not sure if it is by design or a bug, but it do causes some inconvenience from device side. 

BTW, I do not see this problem in cloud-to-device message. 

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. Windows, Linux, OSX]
 - OS Version
 - App version [e.g. 22]

**Additional context**

**Comment by rido-min** [comment]
Hi @xiongyu0523, thank you for reporting this issue. 

Which device SDK are you using? Looking at the MQTT traffic I dont see the quotes.

Can you try with https://mqtt.rido.dev ?

**Comment by xiongyu0523** [comment]
I'm not using a SDK, instead I use micropython umqtt library and I can also see this when I'm using a MQTT debug tool. (MQTTX from EMQ)

In azure iot explorer:

![image](https://user-images.githubusercontent.com/20612136/116960874-41af8c00-acd4-11eb-9958-489ee5bcc71e.png)

In MQTT tool:

![image](https://user-images.githubusercontent.com/20612136/116960887-4ecc7b00-acd4-11eb-9693-ac224fc8f0bc.png)

**Comment by xiongyu0523** [comment]
I'm sure the quotation marks are not from the tool, because if I input 123 in direct method, or I input hello world in C2D, I will not see the quotation

**Comment by rido-min** [comment]
Based on [this doc](https://docs.microsoft.com/en-us/rest/api/iothub/service/devices/invokemethod#request-body) the direct method payload should be JSON.

This issue might be a bug as the tool should not try to accept *no-json-payloads*

/c @montgomp  ?

**Comment by xiongyu0523** [comment]
indeed the document explicitly ask for JSON, but it looks like the backend does not force and you can add any string as payload when invoke a REST API. if iot hub does not limit the format, can Azure IoT Explorer just pass through the content? or we force customer to use JSON on the UI?

**Comment by montgomp** [comment]
I believe this behavior is similar to what is (or was) occurring in the portal as well. Adding to backlog.

**Comment by xiongyu0523** [comment]
Yes I see the same thing in the portal, I believe both are NOT correct. I'd like to see with either fix:
1) remove "" and allow any string in payload as C2D message. 
2) Conform to our doc, and force to use JSON
